### DISCUSSION TITLE
Error when booting

### DISCUSSION THREAD

**Discussion by aldrato** [haugene/docker-transmission-openvpn#2730] [Q&A] [answered]
Hello everyone, my Docker was working fine, but suddenly it started giving the following error during startup. I use Surfshark, and all VPN credentials are valid. Has anyone experienced this issue?

Log:
......
	openvpn/privado/dub-004.ovpn
	openvpn/privado/eze-001.ovpn
	openvpn/privado/eze-002.ovpn
	openvpn/privado/fra-001.ovpn
	openvpn/privado/fra-002.ovpn
	openvpn/privado/fra-003.ovpn
	openvpn/privado/fra-004.ovpn
	openvpn/privado/fra-005.ovpn
	openvpn/privado/fra-006.ovpn
	openvpn/privado/fra-007.ovpn
	openvpn/privado/fra-008.ovpn
	openvpn/privado/gru-001.
Aborting
Updating 45cc9b305..35ec20293

**Comment by ilike2burnthing** [comment, marked as answer]
https://github.com/haugene/docker-transmission-openvpn/issues/2727

**Reply by aldrato** [reply]
Thank you very much! saved my day!

**Comment by aldrato** [comment]
[https://github.com/haugene/docker-transmission-openvpn/issues/2727](url)
